install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -177,6 +182,10 @@ containers:
         fieldPath: metadata.namespace
   - name: ISTIO_META_INTERCEPTION_MODE
     value: "{{ or (index .ObjectMeta.Annotations `sidecar.istio.io/interceptionMode`) .ProxyConfig.InterceptionMode.String }}"
+{{ if contains "*" (annotation .ObjectMeta `traffic.sidecar.istio.io/includeInboundPorts` `DEFAULT_AS_NOT_STAR`) }}
+  - name: ISTIO_META_INBOUND_CAPTURE_ALL_PORT
```

Comment:
There is another PR passing the inbound ports as is.

Could you just pass the value of the annotation ?

Reply:
Thank you for the knowledge. Yes, it there is requirement on passing the ports, it make sense to assume port '*' is indicating capture all port